src/index.js

```diff
@@ -152,8 +152,8 @@ class WarmUp {
     return {
       folderName,
       pathFolder,
-      pathFile: `${pathFolder}/index.js`,
-      pathHandler: `${folderName}/index.warmUp`,
+      pathFile: path.join(pathFolder, 'index.js'),
+      pathHandler: path.join(folderName, 'index.warmUp'),
```

Comment:
`pathHandler` was OK before. Since this is not a path to be used by the OS but a path used by Serverless.

Or am I missing something?

Reply:
This commit may have been heavy handed: I was simply bulk changing paths to get the test suite, which are tun by the git hooks, to allow me to push the changes.